tools/cldr-apps/js/src/esm/cldrTable.js

```diff
@@ -392,7 +389,7 @@ function getSingleRowUrl(tr, theRow) {
   p.append("xpath", theRow.xpathId);
   p.append("fhash", tr.rowHash);
   p.append("automatic", "t");
-  if (cldrStatus.isDashboard()) {
+  if (true || cldrStatus.isDashboard()) {
```

Comment:
leftover?

Reply:
Actually that's the beginning of further work -- updating the dash after the user votes. Currently isDashboard is always false for the dash widget but we need to make the request as though it's true